BMM.UI.Android/Resources/drawable/icon_play.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24"
+        android:viewportHeight="24">
+  <path
+          android:pathData="M7,8.1236V15.8764C7,16.8163 7.7744,17.5 8.628,17.5C8.87,17.5 9.1186,17.445 9.3573,17.3256L17.0969,13.458C18.3008,12.8563 18.3008,11.1436 17.0969,10.5419L9.3573,6.6743C9.118,6.555 8.87,6.5 8.628,6.5C7.7744,6.5 7,7.1837 7,8.1236Z"
+          android:fillColor="#0D131A"
+          android:fillType="evenOdd"/>
+</vector>
```

Comment:
Isn't it the same as in the mini player?

Reply:
I was about to change it but then I found this:
![image](https://user-images.githubusercontent.com/649978/117982048-e4e84d00-b335-11eb-800f-c26f7f10d447.png)